src/Microsoft.Diagnostics.Tracing.Eventing/ProviderConfiguration.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.Diagnostics.Tracing.Eventing
+{
+    public struct ProviderConfiguration
```

Comment:
Would this more accurately be called SessionConfiguration? (or EventPipeSessionConfiguration if we need more context)

A list of providers is part of the session configuration, but then we also have fields such as circularBufferSize, multiFileSec, outputPath, that are solely associated with a session rather than a provider.

Reply:
Makes sense.